dist/win/build.ps1

```diff
@@ -176,10 +176,15 @@ $Env:JP_WIXHELPER_DIR = "."
  "-Dlicense.licenseMergesUrl=file:///$buildDir/../../license/merges"
 
 # download Winfsp
-$winfspMsiUrl= (Select-String -Path ".\bundle\resources\winFspMetaData.wxi" -Pattern '<\?define BundledWinFspDownloadLink="(.+)".*?>').Matches.Groups[1].Value
+$winfspMsiUrl= 'https://github.com/winfsp/winfsp/releases/download/v2.0/winfsp-2.0.23075.msi'
```

Comment:
This script is just for manual building, isn't it? If it is not linked to a specific commit/tag, it might be preferable to use:

```suggestion
$winfspMsiUrl= 'https://github.com/winfsp/winfsp/releases/download/latest/winfsp-2.0.23075.msi'
```

See https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases#linking-to-the-latest-release

Reply:
Nevermind, the asset name contains the version. This won't work...